README.md

```diff
@@ -1 +1,20 @@
 # hw01
+
+
+# About Me
+My name is **Anastasiya Apalkova** and I am a student in the MAPSS program concentrating in psychology. 
+I was born in Ukraine and moved to Chicago with my family when I was seven years old. 
+***
+![Ukraine Map](https://res.cloudinary.com/fen-learning/image/upload/c_limit,w_505,h_490/infopls_images/images/mukraine.gif) ![picture of me](pic4.jpg)
+***
+I did my undergrad at [UW-Madison](https://www.wisc.edu/) where I majored in psychology and sociology. 
```

Comment:
I really liked that you used the link in UW Madison words, instead of just adding the link of the webpage.